### ISSUE TITLE
Using the BasisProject in real world

### ISSUE THREAD

**Issue by GravermanDev** [basisproject/tracker#143]
A few weeks ago, I was contemplating the possibility of creating a project similar to the Basis project. However, my approach was slightly different. Notably, there are existing libertarian socialist groups, such as the Zapatistas in Mexico, which encompass a substantial economy comprising over 200,000 people. Their economy could undoubtedly benefit from digitalization, which would facilitate data collection, enhance economic understanding, and likely serve various other purposes.

I propose that, given the similarity of my idea, we consider collaborating to assist the Zapatistas in becoming the inaugural digitalized decentralized socialist economy. Achieving this objective would necessitate substantial effort, but I believe it can contribute to the realization of the Basis project. I possess some experience with Rust, although it is not extensive.

Here are my recommendations for this project:

 - Contemplate how we can generalize the framework, making it more suitable for a broad spectrum of libertarian socialist, market socialist, decentralized planned, and anarchist projects. We can achieve this by identifying commonalities between them and creating valuable abstractions.
 - After establishing a foundational concept, reach out to the Zapatistas. Inquire about the challenges they perceive that could be resolved through a solution like this and solicit their ideas.
 - Develop it as a "product," with the community as the primary users. Tailor it to their needs, gather feedback, and continuously refine both the concepts and software.
 - Extend our outreach to a broader audience and fine-tune the solution for the Zapatistas, demonstrating its functionality to the community.
 - Ultimately, the Zapatistas can make a democratic decision regarding the adoption of this system.

**Comment by orthecreedence** [comment]
Hey, please forgive me for not responding, I just saw this. I don't know why the issue never got to me in the first place. I'm re-opening it for the time being.

I really like this idea. Reaching out to a similar community for feedback is an excellent way to improve the system. My thoughts on it are complex. This project is specifically geared towards ecology within an industrial productive system. The Zapatistas may have different goals that could change the entire purpose of the project. I think before approaching them, I'd really want to study their economy and understand it. How similar are the two? Do they differ in key ways, or is the overlap significant? I've heard of them for years but I don't really understand how they operate, organize, or produce. It's all kind of foggy to me.

My first inclination is that if you have any contacts for EZLN I'd love to at least start a conversation.

Out of curiosity, what was the project you were thinking of?

**Comment by GravermanDev** [comment]
I agree on the idea of studying the Zapatista economy before reaching out.

Well I am now fully occupied with an experimental project to create a decentralized independent DIY economy inside capitalism, to eventually grow our own food and get out own electricity. It's a extreamly ambitious but we have some prototypes and stuff. Decentralized economic planning with algorithms will probably play a role in this. I can try to help with the idea I gave, but I can't promise much because of all the things I already started.

While writing this I thought for a moment that maybe we could actually use basis, or at least something similar, we will see.

**Comment by orthecreedence** [comment]
That sounds really cool what you're doing. Do you have any ideas for what the decentralized economic planning will look like? And how large is the group you're doing this with?

BTW I'm working pretty heavily on a sister project of Basis called Stamp (https://github.com/stamp-protocol/) which is going to be the backbone of the actual implementation of Basis. [I wrote a bit about the project here.](https://killtheradio.net/technology/stamp-protocol/) The actual [Stamp project website](https://stamp-protocol.github.io/) is horribly out of date and I haven't had time to fix it yet. Once I feel like the network portion of Stamp is in a good place, I'm going to continue implementing the Basis protocol as a p2p system.

**Comment by GravermanDev** [comment]
There is a lot that goes into this so I'll be brief in my explanation. 

First, to even start we need some capital. To get it we will make a worker cooperative and just compete on the market. After that we want to start designing different DIY solutions to produce independently from capitalism. This can be done with hydroponic farming, 3D printers, solar panels and a lot more. We will focus on whatever works for small and local scale. Then we want to start producing, this is the part where we need some decentralized system, not even necessary planning but something needs to be put in place in order to coordinate work.

We are currently designing prototype for our product as a cooperative, so very early into the project.
